cleanlab/filter.py

```diff
@@ -138,6 +139,9 @@ class 0, 1, ..., K-1.
       list of labels for each example, instead of just a single label.
       The multi-label setting supports classification tasks where an example has 1 or more labels.
```

Comment:
```suggestion
      The multi-label setting supports classification tasks where an example can belong to more than 1 class (rather than exactly one class as in standard multi-class classification).
```